packages/preview/project-isi-zac/0.1.0/README.md

```diff
@@ -0,0 +1,59 @@
+# project-isi-zac
+
+A Typst template for writing reports and theses, with sensible defaults and examples.
+
+---
+
+## Getting Started
+
+### Web App
+
+You can start a new project in the Typst web app using the HIT thesis template: https://typst.app/?template=project-isi-zac&version=latest
```

Comment:
I'm not sure "HIT thesis template" is relevant here, shouldn't it just be "the thesis template" or "the template"?